client/src/templates/Challenges/rechallenge/builders.js

```diff
@@ -101,13 +101,18 @@ A required file can not have both a src and a link: src = ${src}, link = ${link}
     .map(source => createBody({ source }))
   );
 
+  /* eslint-disable max-len */
+  const babelPolyfillCDN =
+    'https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.0.0/polyfill.min.js';
+  const babelPolyfill = `<script src="${babelPolyfillCDN}" type="text/javascript"></script>`;
```

Comment:
I came SO close to figuring this solution out myself but couldn't quite get it. Thanks @ValeraS . Good work.